app/controllers/article_controller.rb

```diff
@@ -167,6 +167,16 @@ def eds_init
     config.add_sort_field 'oldest', sort: 'oldest', label: 'date (oldest)'
   end
 
+  def fulltext_link
+    _response, document = search_service.fetch(params[:id])
+    url = extract_fulltext_link(document, params[:type])
+    redirect_to url if url.present?
+  rescue => e
+    Honeybadger.notify(e) if defined? Honeybadger
+    flash[:error] = "Sorry, the PDF download was not successful.\nAlternative route: click the title of the article you want, then try the PDF link on the detail page."
```

Comment:
The `\n` newline character doesn't result in a line break in the HTML.

<img width="883" alt="flash" src="https://user-images.githubusercontent.com/96776/29233022-a322e580-7ea3-11e7-86d3-2199ba896be2.png">